PAO-Project/src/Curs/Materie.java

```diff
@@ -0,0 +1,39 @@
+package Curs;
+
+public class Materie
+{
+    private String denumire;
+    private double nota_s;
```

Comment:
[Fix] in java se foloseste camelCase pentru denumiri de campuri si metode, _ se foloseste doar in constante, care sunt statice si finale